pkg/agent/netpol/network_policy_controller.go

```diff
@@ -51,7 +51,7 @@ const (
 type NetworkPolicyController struct {
 	nodeIP                  net.IP
 	nodeHostName            string
-	serviceClusterIPRange   net.IPNet
+	serviceClusterIPRanges  config.NetIPNets
```

Comment:
This file comes from kube-router; any changes to it would ideally be upstreamed to there first.